model/src/main/kotlin/utils/FindingsMatcher.kt

```diff
@@ -225,3 +225,11 @@ data class FindingsMatcherResult(
 )
 
 private fun TextLocation.directory(): String = path.substringBeforeLast(delimiter = "/", missingDelimiterValue = "")
+
+private fun MutableMap<LicenseFinding, MutableSet<CopyrightFinding>>.merge(
```

Comment:
How about using the existing `zipWithDefault` extension function for maps instead?

Reply:
That'd require turning `val matchedFindings` into `var matchedFindings`.
I'd guess that the current implementation could be more efficient and I find it readable.
Given that I"d prefer to keep it `as-is`. Makes sense to you?